index.js

```diff
@@ -38,12 +38,12 @@ if (!isProduction) {
     app.use(errorhandler());
 }
 
-if (isProduction) {
-    mongoose.connect(process.env.MONGODB_URI);
-} else {
+// if (isProduction) {
```

Comment:
I would rather suggest you remove lines of code that are not being used versus commenting them. Any reason why we have left this here? Do we hope to reuse it sometime in the future for this application?

Reply:
@andela-moe I'll do that, I don't think we'll have need to reuse any of that code.